src/renderer/components/Lofi/Cover/Controls/index.tsx

```diff
@@ -1,22 +1,38 @@
 import * as React from 'react';
 import './style.scss';
-import { API_URL } from '../../../../../constants';
+import { SpotifyApiInstance } from '../../../../../api/spotify-api';
 
 class Controls extends React.Component<any, any> {
+  accountType: string;
   constructor(props: any) {
     super(props);
   }
 
+  componentDidMount() {
+    this.getUserAccountType();
+  }
+
+  async getUserAccountType() {
+    const userProfile = await SpotifyApiInstance.fetch('/me', {
+      method: 'GET',
+      headers: new Headers({
+        Authorization: 'Bearer ' + this.props.token,
+      }),
+    });
+
+    this.accountType = userProfile.product;
+  }
+
   async pausePlay() {
     if (this.props.parent.getPlayState()) {
-      fetch(API_URL + '/me/player/pause', {
+      SpotifyApiInstance.fetch('/me/player/pause', {
         method: 'PUT',
         headers: new Headers({
           Authorization: 'Bearer ' + this.props.token,
```

Comment:
The `Auth` header tends to be re-used a lot. I wonder if we can put it the initial `SpotifyApiInstance` constructor somehow.

Reply:
I agree, I'll move the `Authorization` header in `SpotifyApiInstance`, it will remove some noise from the calling code, good idea!